app/lib/validation/percentage.js

```diff
@@ -6,6 +6,9 @@ module.exports = {
     .messages({
       'any.required': epgErrorMessages.enterEpg,
```

Comment:
Only the `number` keys are required, the Joi validation knows it is only a number.